### PR TITLE
memory: enable tcmalloc for aarch64

### PR DESCRIPTION
**Pull request by rojkov**
Commit Message: memory: enable tcmalloc for aarch64
Additional Description: Also update dependency on Abseil required for newer tcmalloc.
Risk Level: Low
Testing: CI tests
Docs Changes: N/A
Release Notes: updated current.rst
Platform Specific Features: N/A

Contributes to #10053

### CODE REVIEW

**Comment by moderation** [comment]
You might consider bumping Abseil to the latest Oct 29 commit - https://github.com/abseil/abseil-cpp/commit/e9b9e38f67a008d66133535a72ada843bd66013f

**Comment by moderation** [comment]
Dependency updates LGTM. Can approve when this comes out of Draft

**Review by jmarantz — APPROVED** [review]
@envoyproxy/senior-maintainers

**Review by jmarantz — COMMENTED** [review]
Sorry, I just caught this is still in 'draft' mode a std::chrono::millisecond after hitting 'approve' :)

**Comment by rojkov** [comment]
Yeah, I'm still trying to figure out what combination of dep versions builds Ok. :) Hopefully the last commit fixes everything.

**Comment by rojkov** [comment]
@moderation @jmarantz This is ready for review finally.

**Comment by rojkov** [comment]
Abseil cannot be updated to the latest HEAD due to #13973

**Review by lizan — DISMISSED** [review]

**Review by moderation — CHANGES_REQUESTED** [review]
Please update UTC date for abseil-cpp

File: bazel/repository_locations.bzl
Comment (rojkov) [reply bazel/repository_locations.bzl]: Thanks! Updated.

**Review by jmarantz — APPROVED** [review]

**Comment by jmarantz** [comment]
@envoyproxy/senior-maintainers

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**
